match2/wikis/rocketleague/brkts_wiki_specific.lua

```diff
@@ -408,6 +409,27 @@ end
 --
 -- opponent related functions
 --
+function opponentFunctions.getTeamNameAndIcon(template)
+	local team
+	local icon
+	template = (template or ''):lower():gsub('_', ' ')
+	if template ~= '' and template ~= 'noteam' and
+		mw.ext.TeamTemplate.teamexists(template) then
+
+		team = mw.ext.TeamTemplate.raw(template)
+		icon = team.image
+		if icon == '' then
+			icon = team.legacyimage
+		end
+		team = team.page
+	end
+
+	return team, icon
```

Comment:
This could end up being null. Is that handled everywhere?

Reply:
yes, via lines 65 - 67

65: `local name, icon = opponentFunctions.getTeamNameAndIcon(opponent.template)`
(here they might be nul, but if they are they get set via the old functions)

66: `opponent.name = opponent.name or name or opponentFunctions.getTeamName(opponent.template)`
67: `opponent.icon = opponent.icon or icon or opponentFunctions.getIconName(opponent.template)`

Reply:
Oh right, I misread, good 👍